### PR TITLE
Expand field label width over the field's width

### PR DESCRIPTION
**Pull request by frederike-ramin**
# Short Description
- We want labels to be able to go over the complete width of the page while the fields themselves stay at a certain width
- To do this I get rid of the cols attribute and instead set the classes we already set for normal text input fields: input-width-x

# Changes
- Remove cols
- Limit Dropdown width
- Add `flex: 1` to elements inside a `FormRowCentered` to let the elements expand properly
- Add max-width to the `input-width-x` classes
- Remove form row for the request_new_tax_nr field - this was never a row anyways^^

# Feedback
- What do you think of the solution?
- Do you think we also need to do this for the templates right away? Or can that wait until we properly implement them as react components?

### CODE REVIEW

File: webapp/client/src/components/FormFieldDropDown.js [hunk 1]
```diff
@@ -38,13 +38,12 @@ function FormFieldDropDown({
       {...{
         fieldName,
         errors,
-        cols: "6",
       }}
-      labelComponent=<FieldLabel {...{ label, fieldId, details }} />
+      labelComponent={<FieldLabel {...{ label, fieldId, details }} />}
```
Comment (JulianRoesner) [inline webapp/client/src/components/FormFieldDropDown.js:42]: Why did you have to change this?

File: webapp/client/src/pages/TaxNumberPage.js [hunk 2]
```diff
@@ -275,7 +275,7 @@ export default function TaxNumberPage({
   );
 
   const requestNewTaxNumberCheckbox = (
-    <FormRowCentered key="requestNewTaxNumberRow">
+    <>
```
Comment (JulianRoesner) [inline webapp/client/src/pages/TaxNumberPage.js:278]: Why did you remove the FormRowCentered?

**Review by JulianRoesner — COMMENTED** [review]
I like it a lot. However, I do not understand some of the changes.

Besides that, I saw that the `FormFieldConsentBox.js` still uses `cols` in line 62:
```javascript
<FormFieldScaffolding
      {...{
        fieldName,
        errors,
        cols: "12",
```

File: webapp/client/src/components/FormFieldDropDown.js (hunk 1 above)
Comment (frederike-ramin) [reply webapp/client/src/components/FormFieldDropDown.js:42]: My editor showed me that this is a syntax error. More as a warning, I guess (I don't understand Visual Studio Code's visuals yet^^). But I thought as it only makes things better, let's do it #boyscouting

File: webapp/client/src/pages/TaxNumberPage.js (hunk 2 above)
Comment (frederike-ramin) [reply webapp/client/src/pages/TaxNumberPage.js:278]: With the new `flex: 1` on the sub-elements of `FormRowCentered` the items within (heading, text and checkbox) were treated as columns and put next to each other (as items within a row are supposed to) That's why I removed it. They never should have been in a row anyways.
Oh, but is it bad that I removed the key? Is that needed for anyting?

Comment (JulianRoesner) [reply webapp/client/src/pages/TaxNumberPage.js:278]: Ok, but could you re-add the key?
